src/types/io.ts

```diff
@@ -349,6 +349,43 @@ export type AoTokenCostParams = {
   years?: number;
   name: string;
   quantity?: number;
+  fromAddress?: WalletAddress;
+};
+
+export const fundFromOptions = ['balance', 'stakes', 'any'];
+export type FundFrom = (typeof fundFromOptions)[number];
```

Comment:
Does this give exact types and not just `string` without `as const` on fundFromOptions?

Reply:
seems the auto-fill works when setup like his, but the IDE hover shows the type `string`

Reply:
3e76ec4